week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,8 +12,29 @@
  */
 
 class ATM {
-    // Add your code here
-
+  // Add your code here
+  constructor(amount) {
+    this.amount = amount;
```

Comment:
there is a small issue here, your constructure require an argument, but when you created an instance of the class in line 19 you didn't pass it. 
The question will be, do you need the other variable ? amount ?